src/extern.h

```diff
@@ -71,6 +75,8 @@
 
 
 struct conf {
+	int lightcol;	/* use light colors (true/false) */
```

Comment:
I think I would use something like `font_type` instead. I believe it makes more sense since this affects the font in use.